index.html

```diff
@@ -0,0 +1,108 @@
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="nav-bar">
+                <a href="index.html"><img src="./images/Ccsummit2015_logo.png"></a>
+                <ul class="nav-menu">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Program</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                    <li><a href="#">News</a></li>
+                </ul>
+                <button class="campaign"><a href="#">Campaign</a></button>
+            </div>
+            <div class="hamburguer">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
+    <section class="intro">
+        <div class="title">
+            <img src="./images/title.png">
+        </div>
+        <div class="intro-text"> 
+            <h1>A joyful creation believing in the value of openess and sharing, creating a positive change with people
+                over 80 countries is taking place in October, in Korea.</h1>
+        </div>
+        <div class="date-info">
+            <p>2023.05.11(THU) ~ 12(FRI)</p>
+        </div>
+        <div class="location">
+            <p>@RestCasaGrande, Miranda - Venezuela</p>
+        </div>
+    </section>
+    <section class="main">
+    <h2>Main Program</h2>
+    <span class="main-span"></span>  
+    <div class="items">
+        <div class="item">
+        <img src="./images/program_icon_02.png">
+        <h3>Lecture</h3>
+        <p>Small introduction about the meal program and the different menus that will be available on the day of the event</p>
+        </div>   
+        <div class="item">
+        <img src="./images/program_icon_03.png">
+        <h3>Exhibition</h3>
+        <p>Here you will see our best chef working and explaining how each plate of food is made.</p>
+        </div>   
+        <div class="item">
+        <img src="./images/program_icon_04.png">
+        <h3>Tasting</h3>
+        <p>It's time to try the delicacies prepared by our team</p>
+        </div>   
+        <div class="item">
+        <img src="./images/program_icon_05.png">
+        <h3>Feedback</h3>
+        <p>Here we will know your opinion regarding each dish served today</p>
+        </div>
+        <div class="item">
+        <img src="./images/program_icon_01.png">
+        <h3>Dessert</h3>
+        <p>A light dessert before saying goodbye</p>
+        </div>     
+    </div>
+    <a class="program" href="#">SEE THE HOLE PROGRAM</a>
+    </section>
+    <section class="food-section">
+        <p class="featured">Featured Foods</p>
+        <span class="main-span"></span> 
+        <div class="speakers">
+        </div>
+        <div>
+            <button class="more">More<p class="arrow">></p></button>
+        </div>
```

Comment:
- [ ] Nice implementation :+1:  but when your see Behance design there are 2 speakers in one row but in your case, there are 3 speakers so kindly fix this issue also :slightly_smiling_face:  check  below :point_down:  the comparison between your design and Behance design :+1: 

**Your Design**
![Screenshot from 2023-04-20 05-22-58](https://user-images.githubusercontent.com/45226848/233226577-f20db00e-bed3-43ca-aedd-ccef2ac62c4d.png)

**Behance Design**
![Screenshot from 2023-04-20 04-08-59](https://user-images.githubusercontent.com/45226848/233226594-987f329a-b6fd-4004-bc6a-79985327e789.png)